File: app/src/main/java/com/nbennettsoftware/android/npad/NpadDocument.java
package com.nbennettsoftware.android.npad;

import android.net.Uri;

class NpadDocument {

    static final String TYPE_PLAIN_TEXT = ".txt";
    static final String TYPE_NPAD_ML = ".npml";

    private Uri uri;
    private String type;

    NpadDocument(Uri documentUri, String documentType) {
        uri = documentUri;
        type = (documentType != null)? documentType : NpadDocument.TYPE_NPAD_ML;
    }

    Uri getUri(){
        return uri;
    }

    String getType(){
        return type;
    }
}

File: app/src/main/java/com/nbennettsoftware/android/npad/NpadDocument.kt
package com.nbennettsoftware.android.npad

import android.net.Uri

internal class NpadDocument(val uri: Uri?, documentType: String?) {
    val type: String

    init {
        type = TYPE_NPAD_ML
    }

    companion object {

        val TYPE_PLAIN_TEXT = ".txt"
        val TYPE_NPAD_ML = ".npml"
    }
}
